ui/cypress/e2e/explorer.test.ts

```diff
@@ -37,7 +37,7 @@ describe('DataExplorer', () => {
 
     cy.fixture('routes').then(({orgs, explorer}) => {
       cy.get<Organization>('@org').then(({id}) => {
-        cy.visit(`${orgs}/${id}${explorer}`)
+        cy.visit(`${orgs}/${id}${explorer}`, {timeout: 3000})
```

Comment:
it's better to do a combo of `cy.visit` and then use a `cy.get` to see if things have loaded

Reply:
Im going to remove the timeout, i think we dont need it, i just needed it for local testing